xbmc/utils/test/TestSystemInfo.cpp

```diff
@@ -228,9 +227,7 @@ TEST_F(TestSystemInfo, GetUserAgent)
   EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("Windows")) << "'GetUserAgent()' must contain 'Windows'";
 #elif defined(TARGET_DARWIN_IOS)
   EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("like Mac OS X")) << "'GetUserAgent()' must contain ' like Mac OS X'";
-#ifdef TARGET_DARWIN_IOS_ATV2
-  EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("CPU OS ")) << "'GetUserAgent()' must contain 'CPU OS '";
-#else  // ! TARGET_DARWIN_IOS_ATV2
+#ifndef TARGET_DARWIN_IOS_ATV2
```

Comment:
TARGET_DARWIN_IOS_ATV2 is gone 100% (or at least should be). No need to guard it here.